src/library.cpp

```diff
@@ -139,7 +139,7 @@ void Library::loadMonitorDir(QString monitorDir)
     QMutexLocker locker(&mutex);
```

Comment:
This is useless if `mutex` is not shared across `locker`s.
As mutex is here a "simple" variable, each running `loadMonitorDir` has its own mutex and nothing is protected.